scripts/statement.py

```diff
@@ -0,0 +1,15 @@
+from scripts.minmonthpymt import MinMonthPymt
+
+
+class Statement:
+    def __init__(self, balance, minMonthPymtRate):
+        self._balance = balance
+        self._minMonthPymtRate = minMonthPymtRate
+        self._minPmtThisMonth = 0
+
+    def getMinPmt(self, balance, minMonthPymtRate):
+        minPmtThisnMonth = MinMonthPymt(balance, minMonthPymtRate)
+        return minPmtThisnMonth.calcMinMonPymt(minMonthPymtRate, balance)
```

Comment:
Nice, ignorant python question, do you have to pass self to access the things defined in init?

Reply:
yes, you have, I was surprised about this too.